### DISCUSSION TITLE
Quota usage (GB) does not align with percentage

### DISCUSSION THREAD

**Discussion by burnclouds** [nextcloud/all-in-one#2977] [Questions] [answered]
![image](https://github.com/nextcloud/all-in-one/assets/9850267/a978c7e3-852e-4493-ba53-4abbedcfa556)

The quota display shows I'm using 110.8GB of 576.5GB but the bar and hover text indicate 61.84%
Last i checked 110.8/576.5 was 19.22%
Image attached. I verified that my trash was empty.

**Comment by szaimen** [comment, marked as answer]
Hi, please create an issue for this here if there shouldnt be one already: https://github.com/nextcloud/server/issues

**Comment by burnclouds** [comment]
https://github.com/nextcloud/server/issues/39465
